### PR TITLE
docs: Add process "How to assign an issue to someone"

### LINKED ISSUE #14: [DOCS] Add process/guide - "How to assign an issue to someone"
### Changes/Action required
 
 - Add a complete process of **" How to assign an issue to someone"**, steps will contain a text, screenshot, Video (it can be a screen recording to show the steps), and GIF (if needed)
 
- Path of the file in which **steps** need to be added - [`open-source-with-pradumna/pages/How-to/guide/assigning-issue.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How-to/guide/assigning-issue.md)

---

If you have any suggestions feel free to Open an [Issue](https://github.com/Pradumnasaraf/open-source-with-pradumna/issues)

#### This addition of the documentation will get added and hyperlinked in [`How-to/README.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/tree/main/pages/How-to) and also get hosted on the website https://Opensource.pradumnasaraf.co

<br>

Screenshot from 2022-05-15 14-11-21

### PR DESCRIPTION
**Pull request by Prudence97**
## 🛠️ Fixes Issue

closes #14 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey! thank you for opening the PR and putting in your effort.

The issue you are working on is about how to "assign an issue". So the docs should contain procedures only for that. ( I think we don't need to add steps for `opening and labeling the issue` those are different topics, later on, we can hyperlink those docs in this ).

Some more small changes that are required:
- There is no use of `div` tags, please remove them.
- Instead of using **HTML** tags for headings and paragraphs, please use **Markdown syntax.**
- Only add a single blank spaced line.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Pradumnasaraf** [comment]
Hey @Prudence97 waiting for your changes!!

File: pages/How-to/guide/assigning-issue.md [hunk 1]
```diff
@@ -1,4 +1,60 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
+
+
+
+<div>
+  <h1>Creating and assigning an issue from a repository</h1>
+  
+  <p> Issues are important as they alert a maintainer to review and make new changes or receive information. These Information could include bugs, new ideas, feedback etc.
```
Comment (Kanika637) [inline pages/How-to/guide/assigning-issue.md:7]: There are some spelling mistakes which you can take care of like "Information" should be "information".

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Unreadable

File: pages/How-to/guide/assigning-issue.md [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+
+They are also a number of ways to creating an issue from a GitHub repository.
```
Comment (Kanika637) [inline pages/How-to/guide/assigning-issue.md:9]: "They" should be "There"

File: pages/How-to/guide/assigning-issue.md [hunk 3]
```diff
@@ -1,4 +1,60 @@
...
+<h3>Step 3</h3>
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/wvoeblg92h2kgzyhay42.jpg" />
+
+<p> Your issue becomes open after submitting and the issue count increases. The assigned party is also notified.
+
+And, that's one way to create and assign an issue from the GitHub page. </p>
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ozjqbngoagopk1s2slxu.gif" />
+Until later
```
Comment (Kanika637) [inline pages/How-to/guide/assigning-issue.md:60]: I think the steps you have written for the process don't have the required spacing, the steps are getting camouflaged with the screenshots.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by Jai-Solania-29** [comment]
Hey @Prudence97 I went through the changed file and I found many grammatical mistakes, so I would recommend you to double-check once before making a PR.

**Comment by Pradumnasaraf** [comment]
Thank you @Kanika637 for the review. I have made the suggested changes. I think this doc still needs improvement If you want to improve free to raise a PR.

**Review by Pradumnasaraf — APPROVED** [review]
Looks goog to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Reusable

**Pull request merged**